runtime/service.go

```diff
@@ -201,6 +217,7 @@ func (s *service) Create(ctx context.Context, request *taskAPI.CreateTaskRequest
 	}
 	// TODO: should there be a lock here
 	if !s.agentStarted {
+		log.G(ctx).WithField("id", request.ID).Debug("calling startVM")
 		client, err := s.startVM(ctx, request, firecrackerConfig)
 		if err != nil {
 			log.G(ctx).WithError(err).Error("failed to start VM")
```

Comment:
nit: can also include the request.ID here too now

Reply:
Done